### ISSUE TITLE
Automatically open jupyterlab in a browser on container startup

### ISSUE THREAD

**Issue by andrespp** [andrespp/docker-jupyterlab#1]

**Comment by andrespp** [comment]
```console
--browser=<Unicode> (NotebookApp.browser)
    Default: ''
    Specify what command to use to invoke a web browser when opening the
    notebook. If not specified, the default browser will be determined by the
    `webbrowser` standard library module, which allows setting of the BROWSER
    environment variable to override it.
```

**Comment by andrespp** [comment]
Closed by bf1691ee3d
